manifests/template.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
       Example
       libvirt vpx://vcenter.example.com/Datacenter/esxi
```

Comment:
I'*d drop this example because of the one below now.

Reply:
OK